src/components/ArticleList.jsx

```diff
@@ -0,0 +1,36 @@
+import ArticleCard from "./ArticleCard";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import './ArticleList.css'
+
+const endpoint = "https://nc-news-fswv.onrender.com/api/articles"
```

Comment:
Please consider separating out your api requests into their own file -  and also take a look at the axios.create method to make your code even DRYer!

Reply:
Done